app/routes.py

```diff
@@ -1,2 +1,286 @@
-from flask import Blueprint
+import re
+from flask import Blueprint, request, make_response, jsonify
+from flask_sqlalchemy import model
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+import requests
+import os
+from tests.conftest import completed_task
 
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+
+@goals_bp.route("", methods=["GET", "POST"])
+def handle_goals():
+    if request.method == "GET":
+        goals = Goal.query.all()
+        goals_response = []
+        for goal in goals:
+            goals_response.append({
+                "id": goal.goal_id,
+                "title": goal.title
+            })
```

Comment:
List comprehension and a goal helper method would reduce this section into one line. 

```suggestion
        goals_response = [ goal.to_dict() for goal in goals ]
```